source/user/installation_guide.rst

```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
 All the data and configuration needed to run a simulation are located in the `plug`_
 repository, along with usage examples.
 
+Natively
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
++ clone `PowerSimData`_ and `REISE.jl`_ (preferably in the same directly)
```

Comment:
in the same directly -> in the same directory?

Reply:
By the way is it prefearable or mandatory. What happens if you don't do it?

Reply:
I'm not actually sure 🙃 not aware of assumptions in the code, but haven't tested in other configurations. Wouldn't be surprised if nesting the repositories or something could cause weirdness, so wanted to encourage following a known setup.